src/shared/check_outdated_data.c

```diff
@@ -0,0 +1,32 @@
+#include <stdio.h>
```

Comment:
file should be renamed to checkOutdatedData.c like defined in #3

Reply:
okay sir i will do it sorry for mistake i pay 2 rupees